unit-tests/live/CMakeLists.txt

```diff
@@ -8,3 +8,25 @@ dl_file( ${PP_Rosbag_Recordings_URL} recordings recording_deadlock.bag OFF )
 # for rec-play/test-playback-stress.py
 dl_file( ${PP_Rosbag_Recordings_URL} recordings all_combinations_depth_color.bag OFF )
 
+if(TESTDATA_LOCATION)
```

Comment:
What is `TESTDATA_LOCATION`?

Reply:
Fixed. moved the bag files to recordings folder